samples/test-samples.cmake

```diff
@@ -199,7 +206,8 @@ foreach (i RANGE ${len})
   if (UNIX)
     # The file-encryptor and helloworld are special cases which also
     # work under simulation, so we test that additional scenario here.
-    if (${SAMPLE} MATCHES "(file-encryptor|helloworld)")
+    if (${SAMPLE} MATCHES "(file-encryptor|helloworld)" AND NOT DEFINED
```

Comment:
Why is this change needed?

Reply:
I probably pasted it accidentally. I'll remove it later.